.github/classifier.json

```diff
@@ -16,6 +16,7 @@
 		"bracket-pair-guides": {"assign": ["hediet"]},
 		"breadcrumbs": {"assign": ["jrieken"]},
 		"callhierarchy": {"assign": ["jrieken"]},
+		"chat-terminal": {"assign": ["Tyriar"]}
```

Comment:
There seems to be a comma missing:

<img width="748" height="203" alt="Image" src="https://github.com/user-attachments/assets/731f12e9-d6ae-457a-be4c-9808aab97eae" />